packages/arb-token-bridge-ui/src/hooks/useArbQueryParams.tsx

```diff
@@ -177,6 +175,44 @@ export const ChainParam = {
   decode: decodeChainQueryParam
 }
 
+export function encodeTabQueryParam(
+  tab: string | (string | null)[] | null | undefined
+): string {
+  if (!tab || Array.isArray(tab)) {
+    return TabParamEnum[TabParamEnum.BRIDGE].toLowerCase()
+  }
+
+  const tabNumber = Number(tab)
+  if (!Number.isNaN(tabNumber) && tabNumber in TabParamEnum) {
+    return TabParamEnum[tabNumber]!.toLowerCase()
+  }
+
+  return TabParamEnum[TabParamEnum.BRIDGE].toLowerCase()
+}
+
+// Parse string to number
+// URL accept both number and tab name (string)
+export function decodeTabQueryParam(
+  tab: string | (string | null)[] | null | undefined
+): number {
+  if (!tab || Array.isArray(tab)) {
+    return TabParamEnum.BRIDGE
+  }
+
+  const tabUppercased = tab.toUpperCase()
+
+  if (tabUppercased in TabParamEnum) {
```

Comment:
We could sanitize invalid strings server side instead, and default to "bridge" if empty. This way our query params don't have an invalid param and `tab` for "bridge" won't be empty as it's now